package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "miller-columns-element",
-  "version": "0.0.2",
-  "description": "Miller columns element",
+  "version": "0.1.0",
+  "description": "Miller columns (cascading lists) for hierarchical items selection",
```

Comment:
@alex-ju to update this to specify how this is part of GOV.UK taxonomy.

Reply:
Updated. Hope it makes sense.